<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMilestoneDialog</class>
 <widget class="QDialog" name="AddMilestoneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Milestone</string>
  </property>
  <widget class="QLabel" name="milestoneNamelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Milestone name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="startdateTimeEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="totalPeoplelistView">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>140</width>
     <height>305</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="startDateTimelabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date/Time:</string>
   </property>
  </widget>
  <widget class="QListView" name="milestonePeoplelistView">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>140</width>
     <height>305</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertPeoplepushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="teeamres.qrc">
     <normaloff>:/icons/icons/bullet_arrow_right.png</normaloff>:/icons/icons/bullet_arrow_right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removePeoplepushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="teeamres.qrc">
     <normaloff>:/icons/icons/bullet_arrow_left.png</normaloff>:/icons/icons/bullet_arrow_left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="milestoneGrouplabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select milestone group:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="milestoneGroupcomboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectPeoplelabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All people:</string>
   </property>
  </widget>
  <widget class="QLabel" name="workingPeoplelabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;People working at this milestone:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="teeamres.qrc"/>
 </resources>
 <connections/>
</ui>
